dask/dataframe/groupby.py

```diff
@@ -1093,6 +1089,10 @@ def __init__(
         if dropna is not None:
             self.dropna["dropna"] = dropna
 
+        self.sort = {}
+        if sort is not None:
+            self.sort["sort"] = sort
```

Comment:
Maybe I'm missing something, why a dict? I feel like this could be called `groupby_kwargs` or something? But just storing `sort: bool` feels more intuitive to me; I don't see the need for the dict since it'll only ever contain one element.

Reply:
Yeah this is a slightly odd pattern, but one that we've been doing with `dropna` and `observed`. Basically it lets us only pass sort if it is actually set to a bool rather than also passing around `None`. I think I switched to this pattern to make it easier for methods that inherit the kwargs to have their own default for `sort`. If `sort` were being passed as `None` then those defaults need to be handled within the function, but this way they can be handled in the funtiion definition.

Reply:
I see. Could we combine `sort`, `dropna`, and `observed` into one `self.kwargs` then? That would feel a bit more readable to me.

Reply:
Yeah we can definitely do that! I'll revamp this.

Reply:
I mean we can do it, but it won't be used everywhere.